backend/Dockerfile.viewercontroller

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.20.4-alpine3.17 as builder
+FROM golang:1.21.4-alpine3.17 as builder
```

Comment:
some dockerfiles look like they weren't updated like launcher, can you go over those again?

Reply:
done, updated all dockerfiles.